pytorch_pfn_extras/onnx/_as_output.py

```diff
@@ -97,6 +98,8 @@ def trace(
 
 
 def as_output(name: str, value: torch.Tensor) -> torch.Tensor:
+    if torch.jit.is_scripting():
+        return value
```

Comment:
Is it possible to reach here? (I mean, can `as_output` be called during exporing script module?)

Reply:
Calling `as_output` itself is valid for TorchScript. It's in case, I suppose, a user has a module that contains `as_output` calls, and he/she could export the module without removing the `as_output` calls every time. But, yeah, we can remove it if it is uncommon use.

Reply:
I see. Then, how about show a warning in as_output if called from script module saying that the value is no longer an output in the exported onnx?

Reply:
Fine, I'll make it as such!